src/main.ts

```diff
@@ -694,6 +694,20 @@ export class Purchases {
     return this._appUserId;
   }
 
+  /**
+   * Sets attributes for the current user. Attributes are useful for storing additional, structured information on a customer that can be used elsewhere in the system.
+   * For example, you could store your customer's email address or additional system identifiers through the applicable reserved attributes, or store arbitrary facts like onboarding survey responses, feature usage, or other dimensions as custom attributes.
+   * @param attributes - A dictionary of attributes to set for the current user.
+   * @throws {@link PurchasesError} if there is an error while setting the attributes or if the customer doesn't exist.
+   */
+  public async setAttributes(attributes: {
```

Comment:
In the mobile SDKs, we have a `syncAttributesAndOfferingsIfNeeded` that additionally to syncing attributes, it also fetches offerings in case targeting changed... We might want to do something like that here as well... probably less important since we are not caching the value here though.

Reply:
Since we're not caching anything, I think adding this method doesn't add lot of value, and if someone wants to get the offering after setting attributes, they could explicitly call `getOfferings`.

Reply:
Agreed 👍